.github/workflows/evaluate.yml

```diff
@@ -0,0 +1,86 @@
+name: Run evaluation
+
+on:
+  push:
+    branches: [ master, docker ]
+  pull_request:
+    branches: [ master, docker ]
+
+jobs:
+  evaluate:
+    runs-on: ${{ matrix.config.os }}
+    if: "!contains(github.event.head_commit.message, 'ci skip')"
+    
+    strategy:
+      fail-fast: false
+      matrix:
+        config: 
+        - {name: 'current', os: ubuntu-latest, python: '3.8' }
+        - {name: 'prev', os: ubuntu-latest, python: '3.7' }
+        - {name: 'old', os: ubuntu-latest, python: '3.6' }
+
+    steps:
+    - uses: actions/checkout@v2
+    
+    - name: Install system dependencies
+      if: runner.os == 'Linux'
+      run: |
+        sudo apt-get update -qq
+        sudo apt-get install -y libhdf5-dev pandoc gfortran libblas-dev liblapack-dev libedit-dev llvm-dev
+    
+    - name: Install Singularity
+      uses: eWaterCycle/setup-singularity@v3
+      with:
+        singularity-version: 3.6.1
+    
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with: 
+        python-version: ${{ matrix.config.python }}
+
+    - name: Install package & dependencies
+      run: |
+        pip install -U .
+        python -c "import openproblems"
+        pip install -U .[test]
+        black . --check --diff --exclude=website
+    
+    - name: Build docker container
+      run: |
+        # build docker containers using the current repository
+        docker build -f docker/openproblems/Dockerfile -t singlecellopenproblems/openproblems .
+        docker build -f docker/openproblems-r-base/Dockerfile -t singlecellopenproblems/openproblems-r-base .
+    
+    - name: Run tests
+      run: |
+        nose2 openproblems -v
```

Comment:
Does it make sense to split the linting, testing, and evaluation into different actions?